src/terminal/TerminalWindows/BaseTerminalWindow.gd

```diff
@@ -0,0 +1,8 @@
+class_name BaseTerminalWindow
+extends  Resource;
+
+@export var button_text : String;
+var signals : Array;
+
+func get_menu() -> Node:
+	return null;
```

Comment:
shouldn't this return a Control instead? it's a semantic thing, but having it be a control is clearer imo